rules/apple_genrule.bzl

```diff
@@ -80,7 +80,7 @@ def _apple_genrule_impl(ctx):
         inputs = resolved_srcs.to_list() + resolved_inputs,
         outputs = files_to_build.to_list(),
         env = ctx.configuration.default_shell_env,
-        command = argv,
+        command = " ".join(argv),
```

Comment:
should we use the shell escape method in skylib here? https://github.com/bazelbuild/bazel-skylib/blob/182046f090d6ecae264c7e5d520da096370287e2/lib/shell.bzl#L37-L49

Reply:
looks like that overquotes